### DISCUSSION TITLE
리뷰 수락&거절 API 명세

### DISCUSSION THREAD

**Discussion by ohseil** [team-reviewing/reviewing-backend#11] [API 명세]
# 리뷰 수락   

## 기본 정보   
```http
PATCH /reviewers/{reviewer-id}/reviews/{review-id}/status-accepted HTTP/1.1
Host: ${ HOSTNAME }
Authorization: Bearer ${ ACCESS_TOKEN }
```

### Header
|Name|Description|Required|
|-|-|-|
|Authorization|access token|O|

### Path Variable
|Name|Description|Required|
|-|-|-|
|reviewer-id|리뷰어 id|O|
|review-id|리뷰 id|O|

### Request
```http
PATCH hostname/reviewers/{reviewer-id}/reviews/{review-id}/status-accepted
Authorization: Bearer access_token
```

### Response
```http
HTTP/1.1 No Content
```

# 리뷰 거절   

## 기본 정보   
```http
PATCH /reviewers/{reviewer-id}/reviews/{review-id}/status-refused HTTP/1.1
Host: ${ HOSTNAME }
Authorization: Bearer ${ ACCESS_TOKEN }
```

### Header
|Name|Description|Required|
|-|-|-|
|Authorization|access token|O|

### Path Variable
|Name|Description|Required|
|-|-|-|
|reviewer-id|리뷰어 id|O|
|review-id|리뷰 id|O|

### Request
```http
PATCH hostname/reviewers/{reviewer-id}/reviews/{review-id}/status-refused
Authorization: Bearer access_token
```

### Response
```http
HTTP/1.1 No Content
```

**Comment by leeuihyun** [comment]
수정 확인했습니다.

**Comment by ohseil** [comment]
리뷰 거절 기획 변경으로 인해 API 명세 수정되었습니다!
